packages/smooth_ui_library/lib/navigation/smooth_bottom_app_bar.dart

```diff
@@ -0,0 +1,151 @@
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:smooth_ui_library/navigation/models/smooth_bottom_app_bar_item.dart';
+
+class SmoothBottomAppBar extends StatefulWidget {
```

Comment:
In what I had in mind we shouldn't be using something called `SmoothBottomAppBar`. Just a `BottomNavigationBar` that every flutter dev can read.